README.md

```diff
@@ -166,6 +167,35 @@ MESSAGE=> key:user_3, value:LoginCount{userName='user_3', count=1, windowStart=1
 MESSAGE=> key:user_4, value:LoginCount{userName='user_4', count=1, windowStart=1545346145000, windowEnd=1545346146000}
 ```  
 
+6. To open kafka bash run `docker exec -i -t kafka /bin/bash`
+
+Here, you can run Kafka tools from CLI, for example creating a topic
+
+`$KAFKA_HOME/bin/kafka-topics.sh --create --topic my-topic  --zookeeper zookeeper:2181 --partitions 1 --replication-factor 1`
+
+Tow important things to look into in the above command:
+- I use `$KAFKA_HOME` to access Kafka bin tools 
```

Comment:
Not needed. You can use the commands without prefixing with KAFKA_HOME. They are in the path.